buildSrc/build.gradle

```diff
@@ -11,9 +11,10 @@ dependencies {
     compile gradleApi()
     compile localGroovy()
 
-    compile 'org.ajoberstar:gradle-git:1.2.0'
-    compile group: 'org.apache.httpcomponents', name: 'fluent-hc', version: '4.5.2'
-    compile('org.jfrog.buildinfo:build-info-extractor-gradle:4.8.1') {
+    compile 'org.ajoberstar:gradle-git:1.7.2'
+    compile group: 'org.apache.httpcomponents', name: 'fluent-hc', version: '4.5.10'
+    compile('org.jfrog.buildinfo:build-info-extractor-gradle:4.11.0') {
         exclude module: 'groovy-all'
+        exclude module: 'commons-io'
```

Comment:
some comment in here on versions related to publihsing stuff (which should be everything in buildsrc)